src/gcp_scanner/client/client_factory.py

```diff
@@ -56,6 +60,14 @@ def get_client(cls, name):
       return SpannerClient()
     if name == "file":
       return FilestoreClient()
+    if name == "cloudkms":
```

Comment:
Let's keep this sequence of client instantiations sorted alphabetically for readability

Reply:
updated